packages/snaps-utils/src/snaps.ts

```diff
@@ -353,6 +353,21 @@ export function isCaipChainId(chainId: unknown): chainId is string {
   );
 }
 
+/**
+ * Typeguard to ensure an account follows the CAIP-10 standard.
+ *
+ * @param caip10Account - The account being tested.
+ * @returns `true` if the value is a valid CAIP-10 account, and `false` otherwise.
+ */
+export function isCaipAccount(caip10Account: unknown): caip10Account is string {
```

Comment:
Was this needed?

I seem to recall it being mentioned as unused.

Reply:
It was used previously but I removed that code, I'll remove this one too

Reply:
Updated in [18696d5](https://github.com/MetaMask/snaps/pull/1290/commits/18696d5688b5c9148fc3a9b1b77f59f013f50fec)